src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -2088,4 +2127,39 @@ void CodeGen::genX86SerializeIntrinsic(GenTreeHWIntrinsic* node)
     genProduceReg(node);
 }
 
+//------------------------------------------------------------------------
+// genX86SerializeIntrinsic: Generate an "vpmovm2t" instruction, where t
+// represents a byte, word, dword, or qword, from a JIT base type.
+//
+// Operation takes a AVX512 mask register and populates a SIMD register
+// elementwise with 0s or 1s based on the associated bit in the mask register.
+//
+// Arguments:
+//    node - base JIT type
+//
```

Comment:
Method comment needs to be updated

Reply:
I think that method can be removed. Went over the changes again. It's use was removed sometime